libraries/SRV_Channel/SRV_Channel.h

```diff
@@ -49,6 +49,12 @@ class SRV_Channel {
 public:
     friend class SRV_Channels;
 
+    enum class Type {
+        AUX = 16,
+        ANGLE = 17,
+        RANGE = 18,
```

Comment:
Why start at 16?

Reply:
Better than zero :-)  I'd usually wouldn't choose 16, but on this day I did.  Since it's an internally-used enumeration the values don't matter, and if you're stepping through gdb and find the number 17 it's a lot easier to search for that value in the relevant header vs 0 or 1.